src/Task3Test.kt

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+internal class Task3Test {
+
+    @Test
+    //Exceptional case where index -1 is returned
+    // to show that an array of length of less than
+    // 2 was given as input.
```

Comment:
What does this comment even mean ? it is the same for Task2 test1. This method returns bool, never -1?

Reply:
Proves the point that comment should rarely be used as they fall out of sync with the code.

Ideally the code should be so clear that it should not require a comment.